.github/workflows/code_health.yml

```diff
@@ -11,3 +11,13 @@ jobs:
 
       - name: Black Check
         uses: jpetrucciani/black-check@20.8b1
+
+  Mypy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Mypy linting
+        uses: jpetrucciani/mypy-check@master
+        with:
+          path: scripts/*/*.py
```

Comment:
This isn't 100% the same as before.
Before we were using some ugly bash magic to get all python files, we can't do that with this (or other github actions) and we need to either specify the folders containing python files or some magic globbing like this.

A glob that would work would be `**/*.py`  but that syntax breaks github actions :(

Three possibilities here:
1) Trust that all python files will always be in `scripts/*/` or that we will remember to change this 
2) Close this PR and leave mypy checks in evg
3) Create a docker image that works for us

I'm open to both 1 and 2, I won't have time to do 3 for sure :)